File: LV0/주사위 게임 1.js
function solution(a, b) {
  return a % 2 !== 0 && b % 2 !== 0
    ? Math.pow(a, 2) + Math.pow(b, 2)
    : (a % 2 === 0 && b % 2 !== 0) || (a % 2 !== 0 && b % 2 === 0)
    ? 2 * (a + b)
    : a % 2 === 0 && b % 2 === 0
    ? Math.abs(a - b)
    : "";
}

File: LV0/주사위 게임 1.go
func solution(a int, b int) int {
    if a % 2 != 0 && b % 2 != 0 {
        return a * a + b * b
    } else if a % 2 == 0 && b % 2 != 0 || a % 2 != 0 && b % 2 == 0 {
        return 2 * (a + b)
    } else if a % 2 == 0 && b % 2 == 0 {
        return a - b
    }
    return 0
}

